client/toDataByLocation.test.js

```diff
@@ -0,0 +1,87 @@
+const toDataByLocation = require('./toDataByLocation');
+
+test('Testing that an entry wont go through without approval or valid lat and lng', () => {
```

Comment:
nit: more stylistic but I'd remove the "testing that" piece and just describe the test. we already know that this is a test so that's just noise. Something like `test('filters out entries without approval or a valid lat and long', () => ...)`. To break things down even further, I might even have separate tests for filtering out based off the approval vs. invalid lat / lng